src/Okta.Idx.Sdk/Extensions/IdxClientExtensions.cs

```diff
@@ -1,26 +1,26 @@
-﻿// <copyright file="IdxClientExtensions.cs" company="Okta, Inc">
```

Comment:
My IDE decided to "normalize" line endings here.  This file did not actually get modified in any functional way.